config/config.py.template

```diff
@@ -512,6 +516,22 @@ if MLLIB_SPARK_VERSION >= 1.2:
         # (only used for RandomForest)
         OptionSet("feature-subset-strategy", ["auto"])
     ]
+    MLLIB_DECISION_TREE_TEST_OPTS += [
+        # Ensemble type: GradientBoostedTrees
+        OptionSet("ensemble-type", ["GradientBoostedTrees"]),
+        # Path to training dataset (if not given, use random data).
+        OptionSet("training-data", [""]),
```

Comment:
There is some slight redundancy here; we have `--num-trees=1` for both `--ensemble-type=GradientBoostedTrees` and `--ensemble-type=RandomForest`. Is that OK?